src/views/clusteroverview/OverviewTab/running-vms-per-template-card/VMsPerTemplateCard.tsx

```diff
@@ -15,17 +17,22 @@ import {
   TitleSizes,
 } from '@patternfly/react-core';
 
-import { useRunningVMsPerTemplatesHook } from './hooks/useRunningVMsPerTemplateResources';
+import useVMsPerTemplateResources from './hooks/useVMsPerTemplateResources';
 import EmptyStateNoVMs from './utils/EmptyStateNoVMs';
 import RunningVMsChartLegend from './utils/RunningVMsChartLegend';
 import { getChartData, getLegendItems, getTemplateToVMCountMap } from './utils/utils';
 
-import './RunningVMsPerTemplateCard.scss';
+import './VMsPerTemplateCard.scss';
 
-const RunningVMsPerTemplateCard = () => {
+const VMsPerTemplateCard = () => {
   const { t } = useKubevirtTranslation();
-  const useRunningVMsPerTemplatesResources = useRunningVMsPerTemplatesHook();
-  const { loaded, vms, templates } = useRunningVMsPerTemplatesResources();
+  const [activeNamespace] = useActiveNamespace();
+  const namespace = React.useMemo(
+    () => (activeNamespace === ALL_NAMESPACES_SESSION_KEY ? null : activeNamespace),
+    [activeNamespace],
+  );
+  const { loaded, vms, templates } = useVMsPerTemplateResources(namespace);
```

Comment:
maybe useVMsPerTemplateResources can use the active namespace hook? I don't see anywhere else in this file namespace is used, but maybe I'm missing :)

Reply:
Once again sir, you are correct. =) Fixed!